OOP/PrintOutput.java

```diff
@@ -0,0 +1,55 @@
+
+public class PrintOutput {
+
+	
+	
+	
+	public static void showChart(Train.Type type, String fromto) {
+		for (Train train : RailwayReservationSystem.trainList) {
+			if (train.type.equals(type) && train.getFromto().equals(fromto)) {
+
+				System.out.println(train.getName() + "    "
+						+ train.getDuration() + " hrs" + "    "
+						+ train.getAvail());
+
+			}
+
+		}
+
+	}
+
+	public static void showChart(Train.Type type) {
+
+		for (Train train : RailwayReservationSystem.trainList) {
+			if (train.type.equals(type)) {
+
+				System.out.println(train.getName() + "    "
+						+ train.getDuration() + " hrs" + "    "
+						+ train.getAvail() + "    " + train.getFromto());
+
+			}
+
+		}
+
+	}
+
+	public static void printTicket(String name, String trainName,
+			int noOfSeat, int amount) {
+
+		System.out.println("Name  : " + name);
+		System.out.println("train name  : " + trainName);
+		System.out.println("no of seat  : " + noOfSeat);
+		System.out.println("Amount  : " + amount);
+	}
+
+	public static int getAvailable(String trainName) {
```

Comment:
ACTION: If the name of the class is ''PrintOutput', one expects that all it contains is methods which only print. Why should a method which has totally different implementation be there in this class?